include/deal.II/lac/scalapack.h

```diff
@@ -189,6 +189,163 @@ class ScaLAPACKMatrix : protected TransposeTable<NumberType>
                const std::pair<unsigned int,unsigned int> &offset_B,
                const std::pair<unsigned int,unsigned int> &submatrix_size) const;
 
+  /**
+   * Transposing assignment: <i>A = B<sup>T</sup></i>
+   *
+   * The matrices <tt>A</tt> and <tt>B</tt> must have the same process grid.
+   *
+   * Following alignment conditions have to be fulfilled: $MB_A$ = $NB_B$ and $NB_A$ = $MB_B$
```

Comment:
I think that for all of the functions that you're documented, the style could be made a bit more similar to that of the existing document. So here, for example, you could consider writing it as follows:
```
Copy the transpose of the contents of the distributed matrix @p B into this one.

This transpose assignment operation modifies the calling matrix @p A such 
that $A = B^{T}$.

@note The matrices @p A and @p B must have the same process grid.
The following alignment conditions also have to be fulfilled: 
$MB_A = NB_B$ and $NB_A = MB_B$.
```

Reply:
Note also that $MB_A$ = $NB_B$ should be $MB_A = NB_B$ so that the equals is also rendered using LaTeX. Same applies below.

Reply:
I discussed with @davydden  the documentation of `ScaLAPACKMatrix` and we agree that it has to be reworked to make it uniform. I will do that in a different PR based on the style used for the doc in this PR.